defaults/main.yml

```
@@ -10,11 +10,11 @@ aiidalab_server_crypt_key: "{{ lookup('password', '/tmp/crypt_key chars=ascii_le
 # Whether to create new users on first login, if they do not exist
 aiidalab_server_create_users: false  # can be true or false
 
-# Set to true in order to build aiidalab docker image locally
-aiidalab_server_build_locally: false
-
-# Tag of docker stack, see https://hub.docker.com/r/aiidalab/aiidalab-docker-stack
-aiidalab_server_docker_stack: latest  # or 'develop'
+# Manage AiiDAlab Docker image.
+aiidalab_server_build_locally: false # Use `true` to download the Git repository and build the image locally. Use `false` to pull the image from DockerHub.
+aiidalab_server_image_name: aiidalab/aiidalab-docker-stack # Used when pulling the image from DockerHub.
+aiidalab_server_image_git_repository: https://github.com/aiidalab/aiidalab-docker-stack.git # Used when building the image locally from the Git repository.
+aiidalab_server_image_version: latest # or 'develop', or '21.12.0'.
```

One small caveat here is that this variable is used in both cases and needs to be a valid tag.
E.g. `latest` won't exist on the git repo.

I guess we won't need to go into detail

```suggestion
aiidalab_server_image_version: latest # tag of docker image or git repository (e.g. 'latest', 'develop', '21.12.0', ...)
```